ui/apps/platform/src/Components/NotifierConfiguration/NotifierConfigurationForm.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { ReactElement, useState } from 'react';
+import { Button, Card, CardBody, CardTitle, Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import { HelpIcon, PencilAltIcon, PlusCircleIcon, TrashIcon } from '@patternfly/react-icons';
+import { FormikErrors } from 'formik';
+
+import { isDefaultEmailTemplate } from 'Components/EmailTemplate/EmailTemplate.utils';
+import EmailTemplateModal, {
+    TemplatePreviewArgs,
+} from 'Components/EmailTemplate/EmailTemplateModal';
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import useIndexKey from 'hooks/useIndexKey';
+import { NotifierIntegrationBase } from 'services/NotifierIntegrationsService';
+import { NotifierConfiguration } from 'services/ReportsService.types';
+
+import NotifierMailingLists from './NotifierMailingLists';
+
+function splitAndTrimMailingListsString(mailingListsString: string): string[] {
```

Comment:
I noticed this code: `const mailingListsString = mailingLists.join(', ');` inside of `NotifierMailingLists` is making it to where you can't backspace when trying to get rid of a comma. For example, I wouldn't be able to backspace if my input looked like this `test@mail.com, test2@mail.com, ` and my cursor was at the end.

You _could_ do a check on the inputType here to get fix that issue if you think it's worthwhile. Something like this:
```javascript
function splitAndTrimMailingListsString(
    mailingListsString: string,
    event: React.ChangeEvent<HTMLInputElement>
): string[] {
    let adjustedMailingListsString = mailingListsString;
    if (
        adjustedMailingListsString.endsWith(',') &&
        (event.nativeEvent as InputEvent).inputType === 'deleteContentBackward'
    ) {
        adjustedMailingListsString = adjustedMailingListsString.slice(0, -1);
    }
    return adjustedMailingListsString.split(',').map((email) => email.trim());
}
```
Downside being that you have to typecast the nativeEvent and maybe a bit convoluted

Reply:
I have a feeling that might have been something that existed when I initially worked on VM Reports. @pedrottimark we can try to track that as a separate bug if you want.

Reply:
Wow, thank you!

Confirmed as existing problem on staging, so will fix in a follow up contribution.